### DISCUSSION TITLE
Controlling Alert Subject

### DISCUSSION THREAD

**Discussion by aungktun** [nextgenhealthcare/connect#5316] [Q&A] [answered]
Is it possible to change the subject of alert message like this? 

#if(1 == 1 ) WARNING #else ERROR #end

Above velocity code should work but it is not. 

Let me know if conditional statement of Alert subject line. can be done

**Comment by aungktun** [comment, marked as answer]
This works for me in the subject line

#if(${error.toString().contains("blah_blah_blah")}) WARNING #else ERROR #end

**Reply by mjtrac** [reply]
It appears that any variable in the globalMap will be replaced, as will the variables that are in the variable list to the right of the template.  (At first glance, the replacement code might may only be invoked if a $ is present, signalling the presence of a velocity variable needing replacement, and that might explain your 1==1 vs ${something} results, if the replacement is done by running it through velocity.  Check out TemplateValueReplacer in the source.)
